Create my-learnings.md

tarajaneseton opened the pull request:
My Unit 1 homework where I share what I've learned so far on the Black Codher Bootcamp

tarajaneseton committed:
Create my-learnings.md

My Unit 1 homework where I share what I've learned so far on the Black Codher Bootcamp

tarajaneseton committed:
Delete TARA-README

Genyus reviewed (changes requested):
OK, so the content looks good, but you've formatted it as HTML, not Markdown. You should review the [Markdown formatting syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

tarajaneseton committed:
Update my-learnings.md

Changed html tags to md syntax for the first 2 headings, however it doesn't display in the preview.

tarajaneseton commented:
Thank you. I originally wrote it in markdown syntax, but couldn't see any formatting in preview mode. I've just switched the first 2 headings to markdown syntax to demonstrate...

Commit:
Content is now stylised using the Markdown syntax

Genyus commented:
To view a Markdown file, you can simply select "View file" from the three-dots icon at the top of the file diff
<img width="404" alt="Screenshot 2022-09-10 at 02 49 50" src="https://user-images.githubusercontent.com/1282558/189464384-761d6b0a-3ca3-401f-ac5b-a55398302795.png">

To make local checking easier, you can install the [Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) extension, which allows you to view your formatted Markdown files directly in VS Code
